src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -242,11 +242,14 @@ private void doDeleteConsumer(RoutingContext routingContext) {
     }
 
     private Handler<AsyncResult<KafkaConsumerRecords<K, V>>> pollHandler(RoutingContext routingContext) {
+
         return records -> {
-            if (records.succeeded()) {
 
-                TracingHandle tracing = TracingUtil.getTracing();
-                SpanHandle<K, V> span = tracing.span(routingContext, HttpOpenApiOperations.POLL.toString());
+            TracingHandle tracing = TracingUtil.getTracing();
```

Comment:
@ppatierno Hi, My first doubt was regarding this... we want the creation of span inside the `pollHandler` method itself, right?

Reply:
Yes because anyway the `span` instance is used just inside it. It's useless to create outside and pass that in.